tasty/src/dotty/tools/tasty/TastyHeaderUnpickler.scala

```diff
@@ -107,11 +107,20 @@ object TastyHeaderUnpickler {
     }
     val minorVersion = showMinorVersion(MinorVersion, ExperimentalVersion)
     val fileMinorVersion = showMinorVersion(fileMinor, fileExperimental)
-    s"""TASTy signature has wrong version.
-      | expected: {majorVersion: $MajorVersion, minorVersion: $minorVersion}
-      | found   : {majorVersion: $fileMajor, minorVersion: $fileMinorVersion}
-      |
-      |""".stripMargin
+
+    if (MajorVersion == 28 && fileMajor == 28)
```

Comment:
Why do you test `MajorVersion == 28`?

Reply:
I think because 29 would be scala 4, which doesnt match the message